<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QDialog" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип ежемесячных платежей</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="annuitent">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитетные</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="differ">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированные</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sumOfCredit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeCredit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="percent">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
  </widget>
  <widget class="QComboBox" name="monthYear">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Месяцев</string>
   </property>
   <item>
    <property name="text">
     <string>Месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Лет</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="calcDebt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая сумма кредита</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthlyPay">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>205</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="debtPercents">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>235</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalDebt">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>265</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>37</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>97</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>% годовых</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>212</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>242</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>272</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
